The stack protector results in link errors on some older systems.

--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -41,7 +41,7 @@ extra_cxx_compiler_flag("-Wall")
 extra_cxx_compiler_flag("-Wno-missing-braces")
 extra_cxx_compiler_flag("-Wdate-time")
 # Some security flags
-extra_cxx_compiler_flag("-fstack-protector-strong")
+# extra_cxx_compiler_flag("-fstack-protector-strong")
 extra_cxx_compiler_flag("-Wformat")
 extra_cxx_compiler_flag("-Wformat -Werror=format-security")
 extra_cxx_compiler_flag("-D_FORTIFY_SOURCE=2")
